css/style.css

```diff
@@ -1,8 +1,15 @@
 /* font-family: 'Teko', sans-serif; - underdelicious font */
 /* css: font-family: 'Courgette', cursive; */
-html {
+
+* {
+  align-content: center;
```

Comment:
`align-content` is not useful in this context. It only applies if you are using flex-box to organize your elements.